src/celeritas/ext/VecgeomTrackView.hh

```diff
@@ -204,9 +214,13 @@ VecgeomTrackView::operator=(Initializer_t const& init)
     // Note that LocateGlobalPoint sets `vgstate_`. If `vgstate_` is outside
     // (including possibly on the outside volume edge), the volume pointer it
     // returns would be null at this point.
-    detail::BVHNavigator::LocatePointIn(
+#if VECGEOM_VERSION >= 0x020000
+    BVHNavigator::LocatePointIn(
         worldvol, detail::to_vector(pos_), vgstate_, contains_point);
-
+#else
+    BVHNavigator::LocatePointIn(
+        worldvol, detail::to_vector(pos_), vgstate_, contains_point);
+#endif
```

Comment:
These two calls look identical, shouldn't you just remove the `detail::` and it'll work for both cases?